### PR TITLE
Fixed links

### PR DESCRIPTION
**Pull request by jhdalek55**
Moved pointer links from the repo to the web site pages

Please do not merge till I finish. I have one more page to fix.

### CODE REVIEW

File: repositories.md [hunk 1]
```diff
@@ -69,15 +69,15 @@ In order to prevent a new Time Server from accidentally causing a rollback warni
 
 #### Changes to partial verification Secondaries
 
-As partial verification Secondaries only check the Targets metadata from the Director repository, the Time Server key on these ECUs will be checked when verifying the Targets metadata. To do this, check the Time Server key after verifying the most recent Targets metadata file. If the Time Server key is listed in the Targets metadata and has been rotated, reset the clock used to determine the expiration of metadata to a minimal value as described in [Changes to checking Root metadata](#changes-to-checking-root-metadata).
+As partial verification Secondaries only check the Targets metadata from the Director repository, the Time Server key on these ECUs will be checked when verifying the Targets metadata. To do this, check the Time Server key after verifying the most recent Targets metadata file. If the Time Server key is listed in the Targets metadata and has been rotated, reset the clock used to determine the expiration of metadata to a minimal value as described.
```
Comment (trishankatdatadog) [inline repositories.md]: Sorry, as described where though?

Comment (jhdalek55) [reply repositories.md]: I just fixed this. Please wait as I have one more page to correct. I'll be done in a few minutes

Comment (jhdalek55) [reply repositories.md]: Fixed.

**Comment by jhdalek55** [comment]
I'm done now.

**Comment by jhdalek55** [comment]
@trishankatdatadog  I addressed your concern. In editing the text I had left out a few words. Its fixed. If you have no objections, I am merging.

File: repositories.md [hunk 2]
```diff
@@ -69,15 +69,15 @@ In order to prevent a new Time Server from accidentally causing a rollback warni
 
 #### Changes to partial verification Secondaries
 
-As partial verification Secondaries only check the Targets metadata from the Director repository, the Time Server key on these ECUs will be checked when verifying the Targets metadata. To do this, check the Time Server key after verifying the most recent Targets metadata file. If the Time Server key is listed in the Targets metadata and has been rotated, reset the clock used to determine the expiration of metadata to a minimal value as described in [Changes to checking Root metadata](#changes-to-checking-root-metadata).
+As partial verification Secondaries only check the Targets metadata from the Director repository, the Time Server key on these ECUs will be checked when verifying the Targets metadata. To do this, check the Time Server key after verifying the most recent Targets metadata file. If the Time Server key is listed in the Targets metadata and has been rotated, reset the clock used to determine the expiration of metadata to a minimal value as described above.
```
Comment (trishankatdatadog) [inline repositories.md]: Why was the link removed from here, Lois?

Comment (jhdalek55) [reply repositories.md]: Because the link is literally pointing to the paragraph that precedes it. As a result, the link was doing nothing.

**Pull request merged**